src/main/app/forms/models/dateOfBirth.ts

```diff
@@ -27,7 +26,6 @@ export class DateOfBirth implements CompletableTask {
   @ValidateIf(o => o.known === true)
   @ValidateNested()
   @IsValidLocalDate({ message: ValidationErrors.DATE_NOT_VALID })
-  @IsValidYearFormat({ message: ValidationErrors.DATE_INVALID_YEAR })
```

Comment:
why is this removed?

Reply:
The functionality of IsValidLocalDate and IsValidYearFormat have been merged into one now, so no need for both